packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -542,6 +552,26 @@ class CircularProgressIndicator extends ProgressIndicator {
   /// The width of the line used to draw the circle.
   final double strokeWidth;
 
+  /// The shape of the progress indicator's stroke ends.
+  ///
+  /// Determines the appearance of the stroke ends in the progress indicator.
+  /// By default, [strokeCap] is null, causing the stroke ends to be set to
+  /// [StrokeCap.square] when [value] is null (indeterminate), and to
+  /// [StrokeCap.butt] when [value] is not null (determinate).
```

Comment:
What does determinate and indeterminate mean here for folks who may not know?

Reply:
The class already provides it, but I can write it better. Do you have a suggestion? 
I'll try `determinate progress indicator`.
<img width="743" alt="image" src="https://user-images.githubusercontent.com/351125/230979573-14d4cc20-5812-402e-a6ba-cf0106c3cd84.png">